src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
+  const result = [];
```

Comment:
Bad variable name. It is never a good idea to use result. Although checklist doesn't explicitly state that 'result' is a bad variable name. It is presumed that after 'Situs inversus' we should not use such variable names. Better to change this one.

Reply:
Thanks! Agree with you. It's a bad variable name)